src/MediaWiki/Jobs/ChangePropagationDispatchJob.php

```diff
@@ -284,7 +282,7 @@ private function pushChangePropagationDispatchJob( $num, $chunk ) {
 			[
 				'data' => $contents
 			] + self::newRootJobParams(
-				"ChangePropagationDispatchJob:$file:$checkSum"
+				"ChangePropagationDispatchJob:smw_chgprop_$num\_tmp:$checkSum"
```

Comment:
I am wondering why the backslash is there

Reply:
Because a variable can contain an underscore and so it was acting like $num_tmp was a variable.